finalPrep/README.md

```diff
@@ -23,51 +23,54 @@ Some things to consider as you refactor include...
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="938" alt="Screen Shot 2022-01-10 at 2 53 46 PM" src="https://user-images.githubusercontent.com/96446170/148838124-a2e8db2d-aa15-4905-af55-077f9f248b22.png">
 
+- `Add Week 2 Screenshot Here` <img width="1098" alt="Screen Shot 2022-01-10 at 2 53 57 PM" src="https://user-images.githubusercontent.com/96446170/148838167-eb55d976-c2b1-424f-9c17-f35fe3daafd1.png">
+
+- `Add Week 3 Screenshot Here` <img width="1098" alt="Screen Shot 2022-01-10 at 2 54 05 PM" src="https://user-images.githubusercontent.com/96446170/148838195-650fd54e-51b7-4842-a4d5-e4f807708c69.png">
+
+`Calendar Note` In the description section for the HW & Practice, I have specifics on what I plan to work on during that time. 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/Jlavarine/d50267f127ead2d1514a8517bdf7825f
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/Jlavarine/c9c0646fbf5577bea3f9d76bbdfefbdc 
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: https://gist.github.com/Jlavarine/3b9e748bc2bc51af3401a1e4bed9d462
```

Comment:
> I prefer to have less work days by doing more work per day. I enjoy doing larger chunks of work every time I meet with my partners.

I like that you're stating your preferences clearly in this prep for group work. One thing I think you could add is what you want those larger chunks of work to look like, too. How often will you try to take breaks (Pomodoro style? Less structured?). This is a good way to compromise with folks who might prefer shorter group work blocks spread out through the day.

Food for thought before you get into your first of several group DTRs 🙂